src/index.html

```diff
@@ -2,16 +2,137 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            minlength="2"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
```

Comment:
add an empty line between _multiline_ siblings
```suggestion
        </label>
        
        <label class="label">
```